docs/core/advanced_tutorials/using-result-handlers.md

```diff
@@ -0,0 +1,133 @@
+---
+sidebarDepth: 0
+---
+
+# Using Result Handlers
+
+Let's take a look at using result handlers. As introduced in the concept documentation ["Results and Result Handlers"](../concepts/results.md), Prefect Tasks associate their return value with a `Result` object attached to a Task's `State`, and they can be extended with `ResultHandler`s that, when enabled, persist results to storage as a pickle into one of the supported storage backends.
+
+Why would you use result handlers in the first place? One common situation is to allow for inspection of intermediary data when debugging flows (including -- maybe especially -- production ones). By checkpointing data during different data transformation Tasks, runtime data bugs can be tracked down by analyzing the checkpoints sequentially.
+
+## Setting up to handle results
+
+The first step to getting the most out of Result Handlers is to enable results storage. You must enable in two places: globally for the Prefect installation, and at the task level by providing a result handler to tasks either through its flow initialization or as a task-level override. Depending on your storage backend, you may also need to ensure your authentication credentials are set up properly. For Cloud customers, some of this is handled by default.
+
+For Core-only users or Cloud customers on Prefect versions <0.9.1, you must
+- opt-in to checkpointing globally by setting the `prefect.config.flows.checkpointing` to "True".
+- specify the result handler your tasks will use for at least one level of specificity (flow-level or task-level)
+
+For Cloud customers on Prefect version 0.9.1+,
+- checkpointing will automatically be turned on; you can disable it by setting `prefect.config.flows.checkpointing` to "False"
+ - a result handler that matches the storage backend of your `prefect.config.flows.storage` setting will automatically be applied to all tasks , if available; notably this is not yet supported for Docker Storage
+ - you can override the automatic results handler at the global level, flow level, or task level
+
+#### Setting result handler at the Flow level
+```bash
+export PREFECT__FLOWS__CHECKPOINTING=true
+```
+ ```python
+# flow.py
+
+@task
+def add(x, y=1):
+    return x + y
+
+# send the configuration to the Flow object
+with Flow("my handled flow!", result_handler=LocalStorageHandler()):
```

Comment:
This was the only change from the above branch that was a correction: `LocalStorageHandler ` -> `LocalResultHandler`

Reply:
merged in in https://github.com/PrefectHQ/prefect/pull/2076/commits/9f8a32448b4b5535318103ab3167bfb6a3df2216